client/src/global/components/atomic/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+import IconComposer from "global/components/utility/IconComposer";
+import * as Styled from "./styles";
+
+export default function Breadcrumbs({
+  breadcrumbs,
+  backend = false,
+  hideOnDesktop = false
+}) {
+  const Wrapper = backend ? Styled.BackendOuter : Styled.Outer;
+  const Inner = backend ? Styled.BackendInner : Styled.Inner;
+  const Icon = backend ? Styled.Icon : IconComposer;
+
+  return (
+    <Wrapper aria-label="Breadcrumb" $hideOnDesktop={hideOnDesktop}>
+      <Inner>
+        {breadcrumbs.length === 1 &&
+          breadcrumbs.map(crumb => (
+            <Styled.Breadcrumb to={crumb.to} key={crumb.to}>
+              <Icon icon="arrowLeft16" size={backend ? "default" : 24} />
+              {backend && <Icon icon="arrowLeft32" size="default" $desktop />}
+              <Styled.Label>{crumb.label}</Styled.Label>
+            </Styled.Breadcrumb>
+          ))}
+        {breadcrumbs.length > 1 && (
+          <>
+            <Icon icon="arrowLeft16" size={24} />
```

Comment:
I think people are going to be confused if this icon is unclickable. Let's make it part of the first link.